selfdrive/car/honda/carstate.py

```diff
@@ -237,6 +237,7 @@ def update(self, cp, cp_cam, cp_body):
       ret.standstill = cp.vl["ENGINE_DATA"]["XMISSION_SPEED"] < 0.1
       ret.doorOpen = bool(cp.vl["SCM_BUTTONS"]["DRIVERS_DOOR_OPEN"])
     elif self.CP.carFingerprint == CAR.HRV:
+      ret.standstill = not cp.vl["STANDSTILL"]["WHEELS_MOVING"]
```

Comment:
It's probably best to double-check that HRV has STANDSTILL / WHEELS_MOVING signal.  Would the registered HRV test route be suitable?

Reply:
yes, you can use this to pull the rlogs from the test route https://github.com/commaai/openpilot/blob/master/selfdrive/test/openpilotci.py#L10